context/app/static/js/components/detailPage/provenance/utils.ts

```diff
@@ -8,6 +8,18 @@ export function createProvDataURL(uuid: string, entityEndpoint: string) {
   return `${entityEndpoint}/entities/${uuid}/provenance`;
 }
 
+function findDifference<T extends Record<string, unknown>, U extends Record<string, unknown>>(obj1: T, obj2: U) {
+  const diffKeys = Object.keys({ ...obj1, ...obj2 }).reduce((acc, key) => {
```

Comment:
```suggestion
  const diffKeys = Object.keys({ ...obj1, ...obj2 }).reduce<string[]>((acc, key) => {
```
Would this avoid the assertion below?